src/atoms/LinkBarDropdown.js

```diff
@@ -97,6 +98,12 @@ class Dropdown extends React.Component {
 		});
 	}
 
+	tryHide() {
```

Comment:
Please add a comment here, explaining that we often would want a dropdown to go away after clicking a link that is routed client-side.